Description: Include planarity/planarity.h instead of planarity/graph.h
 In newer planarity versions (>= 5.0), the public entrypoint header is
 planarity/planarity.h.
Author: Maximiliano Curia <maxy@debian.org>

Index: sagemath/src/sage/graphs/planarity.pyx
===================================================================
--- sagemath.orig/src/sage/graphs/planarity.pyx
+++ sagemath/src/sage/graphs/planarity.pyx
@@ -3,7 +3,7 @@
 Wrapper for Boyer's (C) planarity algorithm
 """
 
-cdef extern from "planarity/graph.h":
+cdef extern from "planarity/planarity.h":
     ctypedef struct vertexRec:
         int link[2]
         int index
